doc/api.rst

```diff
@@ -69,6 +69,7 @@ checked at compile time in C++20. To pass a runtime format string wrap it in
 Compile-time Format String Checks
 ---------------------------------
 
+Requires C++14 and it's no-op in C++11.
```

Comment:
Let's move this to the end of the paragraph because we should first say what `FMT_STRING` is. Also minor wording tweak:

```
Requires C++14 and is a no-op in C++11.
```

Reply:
Thanks for the brilliant suggestion! It's clearer than before.